### ISSUE TITLE
Using browser back/foward navigation causes json strings to be displayed instead of documents.

### ISSUE THREAD

**Issue by jonaswinkler** [ciur/papermerge#257]
**Description**
I've performed the following actions in the current online demo.
- Open folder "My documents", double click.
- Open the document in that folder, double click.
- Use the browser back navigation.
- User the browser forward navigation.

**Expected**
- The document gets shown again.

**Actual**
- Some json string is shown verbatim, instead of the page.

```
{"document": {"id": 4, "title": "brother_003961.pdf", "notes": "None", "created_at": "30.11.2020 17:49:30", "timestamp": 1606758570.694929, "parent_id": 3, "ctype": "document", "pages": [{"id": 1, "number": 1, "kvstore": []}], "tags": [], "user_perms": {"read": true, "write": true, "delete": true, "change_perm": true, "take_ownership": true}}}
```

**Info:**
 - OS: Archlinux
 - Browser chromium
 - Papermerge Version 1.5.2

**Comment by ciur** [comment]
@jonaswinkler, thank you for opening this bug.
It is very interesting that it can be reproduced only on Chromium browser (I was unable to reproduce it on Firefox/Ubuntu).

**Comment by ciur** [comment]
Fix is now available in master and stable/1.5.x branches.
